pootle/apps/pootle_app/forms.py

```diff
@@ -100,6 +98,51 @@ def clean_code(self):
         return value
 
 
+class ProjectEditForm(ProjectForm):
+    template_tp = forms.ModelChoiceField(
+        label=_('Template tp'),
```

Comment:
Please be aware the TP, Store, Unit are things we talk about in the Pootle internals.  They are not necessarily things we want to expose to users.